examples/demo-stf/src/runtime.rs

```diff
@@ -40,17 +51,17 @@ use sov_modules_macros::{DefaultRuntime, DispatchCall, Genesis, MessageCodec};
 /// Similar mechanism works for queries with the difference that queries are submitted by users directly to the rollup node
 /// instead of going through the DA layer.
 
-#[cfg_attr(feature = "native", cli_parser(DefaultContext))]
+#[cfg_attr(
+    feature = "native",
+    cli_parser(DefaultContext),
+    expose_rpc(DefaultContext)
```

Comment:
Is it possible to remove the `DefaultContext` argument now that this annotation lives on the runtime instead of the STF-Runner?

Reply:
I tried doing this but I got a compilation error saying that `Context` does not implement `DeserializeOwned` which is  required by ``jsonrpsee`. I think we should be able to remove the Context but it will require a bit more digging;